src/js/with-react.jsx

```diff
@@ -53,16 +55,28 @@ const withReact = (WebComponent, { pure = true, passive = false } = {}) => {
     constructor(props) {
       super(props);
 
-      this.handleRef = this.handleRef.bind(this);
-
       this._eventCache = {};
+      this.slot = document.createElement('div');
     }
 
     componentDidMount() {
-      this.componentWillReceiveProps(this.props);
+      this.updateWebComponentProps(this.props);
     }
 
-    componentWillReceiveProps(props) {
+    componentDidUpdate() {
+      const { props, slot, wcNode } = this;
+      const { children } = props;
+      const slotClone = slot.cloneNode(false);
+
+      // @todo: this rebuilds the children's DOM tree every time, NO DIFFING
+      ReactDOM.render(children, slotClone, () => {
```

Comment:
@LucaMele 
This line disables React'S diffing by always creating new children's DOM tree in a temporary `<div>`